examples/labels-2d.py

```diff
@@ -0,0 +1,19 @@
+"""
+Display a labels layer above of an image layer using the add_labels and
+add_image APIs
+"""
+
+from skimage import data
+from skimage.segmentation import slic
+from napari import ViewerApp
+from napari.util import app_context
+
+
+with app_context():
+    astro = data.astronaut()
+    # initialise viewer with astro image
+    viewer = ViewerApp(astronaut=astro, multichannel=True)
+    # add the labels
+    # we add 1 because SLIC returns labels from 0, which we consider background
```

Comment:
I am not too familiar with this space. Is one convention more typical?

Reply:
yes. The whole SLIC-from-zero was a mistake that happened a long time ago and has given us trouble ever since. We'll actually update this at some point.